tests/deepks/Autotest.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 
 # ABACUS executable path
-abacus=/home/fortneu49/abacus-develop/bin/ABACUS.mpi
+abacus=/home/wenfei/abacus-develop/build/abacus
```

Comment:
better turn this default to abacus=abacus

Reply:
sure